module-1/pandas-project/your-code/main.ipynb

```diff
@@ -0,0 +1,4562 @@
+{
```

Comment:
1. Try to print/display only the head of dataframes, this way is better to navigate through your notebook both for you and the people you share with. 
2. It's a good practice to write down a brief description of your dataset and the goals of your analysis at the beginning of the exercise. This facilitates the comprehension of your code. 
3. Why are you using set in many cells of the notebook? It doesn't make sense at all.
4. It's very useful for you to try different approaches to get the results you want but please use other files to play. Leave the notebook in the pull request as clean as possible. 
5. Remember to change the cell type in your notebook for notations. It should be markdown cells otherwise the code throws an error. 
6. In[140]: What commented this afternoon, bins are groups of records under certain condition you previously defined. The best example of this is when you plot an histogram. You only can build bins for numeric or categorical values. We get through this on tomorrow's 15 minutes meeting if you want.  
7. In[142]: What do you want to do in this cell? "df["Player"].astype(str).astype(int128)". It's quite confusing for me.